docs/plugins.rst

```diff
@@ -128,3 +128,30 @@ How to use it
     };
 
     Voxa.plugins.cloudwatch(skill, cloudWatch, eventMetric);
+
+
+
+Autoload plugin
+------------------
+
+It accepts an adapter to autoload info into the model object coming in every alexa request.
+
+Params
+******
+
+.. js:function:: autoLoad(skill, [config])
+
+  Autoload plugin uses ``skill.onRequestStarted`` to load data the first time user open a skill
```

Comment:
actually, using onRequestStarted means the data is loaded every time the user makes a requests to alexa, not only the first time when the user starts the session